src/main/java/org/helpberkeley/memberdata/WorkflowExporter.java

```diff
@@ -40,7 +41,8 @@ public class WorkflowExporter extends Exporter {
     private final WorkflowParser parser;
     private final List<WorkflowBean> updatedBeans = new ArrayList<>();
     private final Set<String> updatedUsers = new HashSet<>();
-    private int member_limit = Constants.AVG_RUN_SIZE*10;
+    public static int getDefaultMemberLimit() { return Constants.AVG_RUN_SIZE*10; }
+    public static int member_limit = getDefaultMemberLimit();
```

Comment:
Minor nit.   One of the style aspects of the code in this repository is that class CTOR definition(s) come first, before other method definitions  (though I wouldn't be surprised to find a few thoughtless exceptions in the project).  Maintaining a consistent style within a project makes it easier to read, learn, and maintain it.  In software organizations/projects there are generally coding standards that must be met.

Here, I would either move getDefaultMemberLimit() next to changeMemberLimit(), or yet simpler, just define it as a static final constant:
```public static final int DEFAULT_MEMBER_LIMIT = Constants.AVG_RUN_SIZE * 10 ;```

Another style point is that [camelCase](https://en.wikipedia.org/wiki/Camel_case) naming is used throughout this project. Change this to to memberLimit;

Change memberLimit to be private, and add an accessor for it to be used by the test case.